core/src/service/file_service.rs

```diff
@@ -1908,10 +1896,10 @@ mod unit_tests {
             .unwrap();
         file_service::prune_deleted(config).unwrap();
 
-        assert_metadata_changes_count!(config, 1);
+        assert_metadata_changes_count!(config, 2);
         assert_document_changes_count!(config, 0);
         assert_metadata_count!(config, RepoSource::Base, 2);
-        assert_metadata_count!(config, RepoSource::Local, 2);
```

Comment:
because a move for a deleted document is no longer undone, the document still has the folder as a parent, so the folder is not pruned